### DISCUSSION TITLE
When will a newly added language start being detected?

### DISCUSSION THREAD

**Discussion by davidbrown2324** [github-linguist/linguist#5215] [Classification] [answered]
I recently added ImageJ Macro language saved in .ijm files to linguist (Thank you). 

I have since updated repos containing .ijm files, but they do not register as ImageJ Macro.

In case this is due to the relative size of the files, I made a [test repo](https://github.com/davidbrown2324/ijm_test_repo) that contains the same .ijm file as the sample ImageJ Macro file added to linguist.

This does not registers as ImageJ Macro, even though it is the only code file in the repo.

Am I missing something?

**Comment by lildude** [comment, marked as answer]
> Am I missing something?

Yes. I've not made and deployed a new release yet 😁  New languages and updates don't instantly appear on GitHub.com. I need to make and deploy a new release.

I've started making a new release in https://github.com/github/linguist/pull/5209 and as per my comment at https://github.com/github/linguist/discussions/5197#discussioncomment-359076 I plan to make the release early this week.

**Reply by davidbrown2324** [reply]
Okay, thank you. That makes sense. I am learning so much.

**Comment by lildude** [comment]
All done and now live on GitHub.com and Lightshow!! 🎉  You will need to push changes to force re-analysis.

**Reply by davidbrown2324** [reply]
It's beautiful. Thanks.
